src/routes/sunshine.js

```diff
@@ -94,8 +95,54 @@ module.exports = (app, db) => {
         res.status(200).json(db.getFields());
     };
 
+    const exportHandler = async (req, res) => {
+        const query = queryFromReq(req);
+
+        // headers - array of strings
+        const headersToCSV = (headers) => {
+            var line = '';
+            for (header of headers) {
+                line += '"' + header.toString() + '",';
+            }
+            line += '\n';
+            return line;
+        }
+
+        // entry - an object from database to convert to a CSV line
+        // headers - array of headers
+        const entryToCSV = (entry, headers) => {
+            var line = '';
+            for (header of headers) {
+                if (entry[header] === undefined) {
+                    line += ',"",';
+                } else {
+                    line += ('"' + entry[header].toString() + '",');
+                }
+            }
+            line += '\n';
+            return line;
+        }
+
+        try {
+            const cursor = await db.getCursor(query);
+            const csvHeaders = db.getFields();
+            res.setHeader('Content-disposition', 'attachment; filename=SundialExport.csv');
```

Comment:
I don't know if these response headers are correct, but it does work with them so they work. Just want to make sure you've checked to ensure these are all that are required for the request to be "proper".